blog/_posts/TeamDelSquared.md

```diff
@@ -0,0 +1,40 @@
+### Team DelSquared: Our Summer of Code, 2018
+
+ **And...it’s a wrap!**
+ 
+September is approaching to an end and we still can’t believe how fast the last three months have gone by. This is Arushi and Sharmistha, from India, and we worked this summer on the Tessel project. The experience has been simply wonderful. Sitting for hours trying to figure out how to go about our first pull request, the weekly and monthly meetings with our team, closing the first issue we worked on, receiving the package with Tessel modules (and stickers of course!!!) - these are a few of the many instances that have made our summer a truly memorable experience.
```

Comment:
Please breakdown whole paragraph in to chunks of text and add blank lines in between. The whole long paragraph makes it hard to read.